platform/core/src/services/HangingProtocolService/HangingProtocolService.ts

```diff
@@ -99,7 +127,11 @@ class HangingProtocolService {
 
     // if there is no pre-defined protocol
     if (!protocol || protocol.id === undefined) {
-      const matchedProtocol = this.ProtocolEngine.run(metaData);
+      const matchedProtocol = this.ProtocolEngine.run({
+        studies: this.studies,
+        studyMetaData: studies[0],
```

Comment:
There was a reason that we copied the studyMetadata with Object.assign, please revert/include the copy instead of direct reference

Reply:
If you are passing both `studies` and `studyMetadata` (Metadata instead of MetaData - to be consistent with cs3d onwards), then can we name `studyMetadata` as `activeStudyMetadata`?

Reply:
The reason for the object copy was that the object was modified internally.  That is no longer happening as the matcher creates a new referenced object containing just the values being matched, so it isn't necessary to have an extra copy.  If you want an extra copy of that, then the display sets should also be fully duplicated everywhere they get accessed.  It also wasn't a fully hierarchical copy, so the old code incorrectly modified the series data.  It would be really nice to have a readonly attribute on the value.